test/e2e-v2/cases/rocketmq/docker-compose.yml

```diff
@@ -0,0 +1,117 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: "3"
+
+services:
+  oap:
+    extends:
+      file: ../../script/docker-compose/base-compose.yml
+      service: oap
+    ports:
+      - "12800:12800"
+    networks:
+      - e2e
+
+  rocketmq-namesrv-1:
+    image: apache/rocketmq:4.9.4
+    hostname: rocketmq-namesrv-1
+    ports:
+      - "9876:9876"
+    environment:
+      TZ: Asia/Shanghai
+    command: sh mqnamesrv
+    healthcheck:
+      test: [ "CMD", "nc", "-nz", "127.0.0.1", "9876" ]
+      interval: 15s
+      timeout: 10s
+      retries: 120
+    networks:
+      - e2e
+
+  rocketmq-broker-1:
+    image: apache/rocketmq:4.9.4
+    hostname: rocketmq-broker-1
+    ports:
+      - "10911:10911"
+      - "10909:10909"
+    environment:
+      TZ: Asia/Shanghai
+      NAMESRV_ADDR: "rocketmq-namesrv-1:9876"
+    command: sh mqbroker
+    healthcheck:
+      test: [ "CMD", "nc", "-nz", "127.0.0.1", "10911" ]
+      interval: 15s
+      timeout: 10s
+      retries: 120
+    depends_on:
+      rocketmq-namesrv-1:
+        condition: service_healthy
+    networks:
+      - e2e
+
+  rocketmq-producer-perf-test:
+    image: apache/rocketmq:4.9.4
+    hostname: rocketmq-producer-perf-test
+    environment:
+      TZ: Asia/Shanghai
+      NAMESRV_ADDR: "rocketmq-namesrv-1:9876"
+    command: sh tools.sh org.apache.rocketmq.example.quickstart.Producer
+    depends_on:
+      rocketmq-broker-1:
+        condition: service_healthy
+    networks:
+      - e2e
+
+  rocketmq-consumer-perf-test:
+    image: apache/rocketmq:4.9.4
+    hostname: rocketmq-consumer-perf-test
+    environment:
+      TZ: Asia/Shanghai
+      NAMESRV_ADDR: "rocketmq-namesrv-1:9876"
+    command: sh tools.sh org.apache.rocketmq.example.quickstart.Consumer
+    depends_on:
+      - rocketmq-producer-perf-test
+    networks:
+      - e2e
+
+  rocketmq-exporter:
+    image: hfuiewiu/rocketmq-exporter:latest
```

Comment:
Does this have an official image? 
And please use a specific version instead latest.

Reply:
> Does this have an official image? And please use a specific version instead latest.

It hasn't published an official image, i will tag a new specific version intead latest

Reply:
> Does this have an official image? And please use a specific version instead latest.
```rocketmq-exporter``` hasn't published a released version. Is it okay if I name it version 0.1?"

Reply:
Who own this repository? Does not RocketMQ upstream support metrics exporter?

Reply:
> Does not RocketMQ upstream support metrics exporter?

This repo owner is me , this exporter is official, but it hasn't pulished a released image

Reply:
> Who own this repository? Does not RocketMQ upstream support metrics exporter?

This repo owner is me , this exporter is official, but it hasn't pulished a released image

Reply:
Why upstream doesn't export an official image? Is there any block?

Reply:
> rocketmq-exporter hasn't published a released version. Is it okay if I name it version 0.1?"

In the pom the latest code is 0.0.2-SNAPSHOT https://github.com/apache/rocketmq-exporter/blob/72ce20c7983693e54f74075d68a9f04aa32c8644/pom.xml#L13C14-L13C28

Reply:
If the RocketMQ could release a version, SkyWalking side should be good.

Could you discuss this with them?

Reply:
> Why upstream doesn't export an official image? Is there any block?

This repo has been active for 5 years. It seems there are no blocks. The RocketMQ official website has introduced this feature（https://rocketmq.apache.org/docs/4.x/deployment/04Exporter）, but the maintainer hasn't released it

Reply:
> > rocketmq-exporter hasn't published a released version. Is it okay if I name it version 0.1?"
> 
> In the pom the latest code is 0.0.2-SNAPSHOT https://github.com/apache/rocketmq-exporter/blob/72ce20c7983693e54f74075d68a9f04aa32c8644/pom.xml#L13C14-L13C28

Ok,i forgot it

Reply:
> If the RocketMQ could release a version, SkyWalking side should be good.
> 
> Could you discuss this with them?

I could have a try

Reply:
> > Why upstream doesn't export an official image? Is there any block?
> 
> 
> 
> This repo has been active for 5 years. It seems there are no blocks. The RocketMQ official website has introduced this feature（https://rocketmq.apache.org/docs/4.x/deployment/04Exporter）, but the maintainer hasn't released it

This is not a hard requirement, but we still prefer this is an official release. Main reasons are
1. Personal redistribution may have brand conflict if you call that RocketMQ exporter release.
1. End users may confuse and are not willing to use.

Reply:
> > > rocketmq-exporter hasn't published a released version. Is it okay if I name it version 0.1?"
> > 
> > 
> > In the pom the latest code is 0.0.2-SNAPSHOT https://github.com/apache/rocketmq-exporter/blob/72ce20c7983693e54f74075d68a9f04aa32c8644/pom.xml#L13C14-L13C28
> 
> Ok,i forgot it

Thanks for your contribution, 

> > > rocketmq-exporter hasn't published a released version. Is it okay if I name it version 0.1?"
> > 
> > 
> > In the pom the latest code is 0.0.2-SNAPSHOT https://github.com/apache/rocketmq-exporter/blob/72ce20c7983693e54f74075d68a9f04aa32c8644/pom.xml#L13C14-L13C28
> 
> Ok,i forgot it

It's a great job, will greatly improve the observability of RocketMQ servers, 
RocketMQ community is discussing the release of an official version in the mailing list, and I'm willing to push this forward.  welcome to get involved in this discussion. Last, thanks for your contribution ：）

Reply:
@peachisai Could you track that discussion? We could continue this after the release is done.

Meanwhile, you could send another pull request to land the SWIP-3. And you could prepare the showcase part for this.

Reply:
> > > > rocketmq-exporter hasn't published a released version. Is it okay if I name it version 0.1?"
> > > 
> > > 
> > > In the pom the latest code is 0.0.2-SNAPSHOT https://github.com/apache/rocketmq-exporter/blob/72ce20c7983693e54f74075d68a9f04aa32c8644/pom.xml#L13C14-L13C28
> > 
> > 
> > Ok,i forgot it
> 
> Thanks for your contribution,
> 
> > > > rocketmq-exporter hasn't published a released version. Is it okay if I name it version 0.1?"
> > > 
> > > 
> > > In the pom the latest code is 0.0.2-SNAPSHOT https://github.com/apache/rocketmq-exporter/blob/72ce20c7983693e54f74075d68a9f04aa32c8644/pom.xml#L13C14-L13C28
> > 
> > 
> > Ok,i forgot it
> 
> It's a great job, will greatly improve the observability of RocketMQ servers, RocketMQ community is discussing the release of an official version in the mailing list, and I'm willing to push this forward. welcome to get involved in this discussion. Last, thanks for your contribution ：）

Thank you for the promotion. I would like to join the discussion.

Reply:
> @peachisai Could you track that discussion? We could continue this after the release is done.
> 
> Meanwhile, you could send another pull request to land the SWIP-3. And you could prepare the showcase part for this.

Okay, I will follow up on this matter.If there is any progression, I will post it on SWIP-3.